docs/overview.md

```diff
@@ -0,0 +1,56 @@
+# Charmed Apache Kyuubi K8s
+
+Charmed Apache Kyuubi K8s, an open-source [Kubernetes charm](https://juju.is/docs/olm/charmed-operator) for Apache Kyuubi, is a distributed and multi-tenant gateway to provide serverless SQL on Data Warehouses and Lakehouses built with the Charmed Apache Spark solution.
+
+Charmed Apache Kyuubi K8s provides users with a JDBC-compliant endpoint which can be used by JDBC-compliant tools and analytics platforms to execute SQL queries on the Charmed Spark cluster.
+It is part of an open source, end-to-end, production-ready data platform on top of cloud native technologies provided by Canonical.
```

Comment:
I'm not sure whether we need this part but if we do need it, consider adding some link(s) to it. Maybe [Spark](https://canonical.com/data/docs/spark/k8s/t-overview) or at least https://canonical.com/data

Reply:
Added in f134856e2cf10061a8035d776c34210c0ffedd00. I think the initial meaning of this part was to emphasize that this charm is not self-sufficient